svarut-rest-klient/src/main/java/no/ks/fiks/svarut/klient/SvarUtKlientApiImpl.java

```diff
@@ -279,6 +278,46 @@ public List<String> forsendelseTyper() {
         }
     }
 
+    @Override
+    public List<MottakerForsendelsesTyper> retrieveMottakersystem(String organisasjonsNummer, String forsendelsesType, Integer niva) {
+        final Request request = client.newRequest(baseUrl + "/tjenester/api/forsendelse/v1/mottakersystem");
+
+        if (organisasjonsNummer != null) {
+            if (!organisasjonsNummer.matches("^[0-9]{9}$")) {
```

Comment:
Har du vurdert å bruke ny validering av orgnr

Reply:
Konkluderte med at det er out-of-scope for denne oppgaven